tutorials/cp-starter-integration-cpi-onboard-subscribe/cp-starter-integration-cpi-onboard-subscribe.md

```diff
@@ -1,79 +1,98 @@
 ---
 title: Set Up Your SAP Cloud Platform Integration Tenant
-description: Subscribe to the Process Integration service, assign required roles, and use the self-service application to provision your own SAP Cloud Platform integration tenant.
+description: Subscribe to the Process Integration service, assign required roles, and use the self-service application to provision your own SAP Cloud Platform Integration tenant.
 auto_validation: true
 time: 15
 tags: [ tutorial>beginner, products>sap-cloud-platform, products>sap-cloud-platform-connectivity, products>sap-cloud-platform-for-the-cloud-foundry-environment]
 primary_tag: products>sap-cloud-platform-integration-for-process-services
+author_name: VIKRAM KULKARNI
+author_profile: https://github.com/Vikramkulkarni01
 ---
 
 ## Details
 ### You will learn
   - How to subscribe to the Process Integration service
   - How to set up your own SAP Cloud Platform Integration tenant
 
-To provision your SAP Cloud Platform Integration tenant, you must first subscribe to the process integration service, assign the necessary roles to your user, and then provision your SAP Cloud Platform Integration tenant.
+###Prerequisites
+To provision your SAP Cloud Platform Integration tenant, you must first subscribe to the Process Integration service, assign the necessary roles to your user, and then provision your SAP Cloud Platform Integration tenant.
 
 ---
 
-[ACCORDION-BEGIN [Step 1: ](Subscribe to Process Integration service)]
-1. Login to the [SAP Cloud Platform Cockpit](https://cockpit.hanatrial.ondemand.com/cockpit/#/home/trialhome) and choose **Enter your Trial Account**. If this is your first time accessing the trial account, the cockpit will create a new trial account for you. Please wait for the process to complete and then click on **Enter your Trial Account** again.
+[ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-    ![Access trial account](1.0.0.create-cp-subaccount.png)
+1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
 
-    You see a subaccount with name **trial** already created. Select this subaccount.
+2. Navigate to the trial global account by clicking **Enter Your Trial Account**.
 
-    ![Access subaccount](1.0.1.access-subaccount.png)
+    !![Trial global account](01_Foundation20Onboarding_Home.png)
```

Comment:
This image is enormous -- I know Annika or someone else may have supplied. I would suggested cutting after the Quick Access Tool section.

Reply:
@thecodester, 
Corrected image.